infracost-usage-example.yml

```diff
@@ -739,6 +739,19 @@ resource_usage:
     throughput_or_capacity_units: 10  # Number of Throughput Units (for Basic and Standard) and Capacity units (for Dedicated) namespaces.
     capture_enabled: false            # Defines if capture is enabled for the Event Hub Standard namespaces, can be: true, false.
 
+  azurerm_frontdoor.my_frontdoor:
+    monthly_outbound_data_transfer_gb: # Monthly outbound data transfer from the following, in GB:
+      north_america_europe: 200000         # North America, Europe and Africa
```

Comment:
The Pricing Calculator calls the Zone1 "North America, Europe and Africa". Using `north_america_europe` for a shorte name.

Reply:
I would go with `north_america_europe_africa` or `zone_1`. @alikhajeh1 any opinions?

Reply:
I guess for the usage file `north_america_europe_africa` is more user-friendly. Although it doesn't include ALL regions in these names. Maybe we could also add a comment with a link to calculator, so if not sure users will have a quick way to find it?

Reply:
+1 on `north_america_europe_africa` as it's self-documenting.